app/helpers/recommendations_helper.rb

```diff
@@ -66,26 +66,32 @@ def recommendations_sites
     {
       "stop-the-tories" => {
         order: 1,
+        name: "Stop The Tories",
         link: "https://stopthetories.vote/"
       },
       "sprint-for-pr" => {
         order: 2,
+        name: "Sprint for PR",
         link: "https://sprintforpr.uk/plan"
       },
       "tactical-vote" => {
         order: 3,
+        name: "Tactical (dot) Vote",
```

Comment:
This `(dot)` looks a bit weird to me TBH.  How about `Tactical.Vote`?  I do appreciate the need to make it substantially different from `Tactical Vote UK` but hopefully this is different enough?

Reply:
That was @tdg 's suggestion.

Reply:
Gotcha.  @tdg can we think of something a bit less weird looking here please? :stuck_out_tongue_winking_eye: